src/AltinnCore/ServiceLibrary/Models/PDFContext.cs

```diff
@@ -23,13 +23,13 @@ public class PDFContext
         /// Gets or sets the form layout
         /// </summary>
         [JsonProperty(PropertyName = "formLayout")]
-        public JObject FormLayout {get; set;}
+        public object FormLayout {get; set;}
```

Comment:
Could we find another way to solve the problem instead of using object?

Reply:
Out of scope